Update interfaces/feature-policy.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/e0218fe/whatwg/idl/feature-policy.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/506822321

foolip commented:
See https://github.com/web-platform-tests/wpt/pull/14956 for what's going on here.

foolip commented:
I now consider this blocked on https://github.com/tidoust/reffy/issues/142.

autofoolip committed:
Update interfaces/feature-policy.idl

Source: https://github.com/tidoust/reffy-reports/blob/e0218fe/whatwg/idl/feature-policy.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/506822321
